app/consumer/export.go

```diff
@@ -52,22 +51,22 @@ func (app *App) ExportAppStateAndValidators(
 // NOTE zero height genesis is a temporary feature which will be deprecated
 //      in favour of export at a block height
 func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []string) {
-	applyAllowedAddrs := false
+	// applyAllowedAddrs := false
```

Comment:
+1 I'd rather avoid commenting if possible.

Reply:
+1 I'd rather avoid commenting if possible.

Reply:
Yes since it's only read to populate the staking module states here
https://github.com/cosmos/interchain-security/blob/c3a329be764fbb28ecd28a24f502d7db6e20ee5c/app/consumer/export.go#L159-L163